### PR TITLE
feat: more updated docker instructions

### PR DESCRIPTION
**Pull request by c-ehrlich**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-08-15).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

A few things were still missing in https://github.com/t3-oss/create-t3-app/pull/333
- next.config.mjs links to the file in the repo in a few places
- some suggested changes were accepted but not copied from readme to docs
- a link in docker-compose had whitespace in the middle of it

### CODE REVIEW

**Comment by kyuuaria** [comment]
Isn't `[next.config.mjs](./next.config.mjs)` fine in the readme?

File: cli/template/base/README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Please note that Next.js requires a different process for buildtime (available i
      });
    ```
 
-2. Remove the `env`-import from [next.config.mjs](./next.config.mjs):
+2. Remove the `env`-import from [next.config.mjs](./template/base/next.config.mjs):
```
Comment (juliusmarminge) [inline cli/template/base/README.md:58]: This link isn't correct?

**Review by juliusmarminge — COMMENTED** [review]
Sorry for merging too soon xD

File: cli/template/base/README.md (hunk 1 above)
Comment (juliusmarminge) [reply cli/template/base/README.md:58]: Sorry for all the suggestions, but when I think of it, we could merge steps 1 and 2? Using diff-mode makes that quite handy, what's a think?

Comment (juliusmarminge) [inline cli/template/base/README.md:58]:
```suggestion
2. Remove the `env`-import from [next.config.mjs](./next.config.mjs):
3. ```

**Pull request merged**